### PR TITLE
Add ue-rs to image

### PR DESCRIPTION
**Pull request by jepio**
# Add ue-rs to image

<to be filled by @wrl>

## How to use

[ describe what reviewers need to do in order to validate this PR ]

## Testing done

[Describe the testing you have done before submitting this PR. Please include both the commands you issued as well as the output you got.]

- [ ] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [ ] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.

### CODE REVIEW

File: sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-0.1.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1 @@
+ue-rs-9999.ebuild
```
Comment (krnowak) [inline sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-0.1.0-r1.ebuild:1]: I'm not sure if having `ue-rs-9999` makes any sense. Could we have `ue-rs-0.1.0-r1.ebuild` as a regular file and drop the `9999` one?

File: sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+	windows_x86_64_msvc-0.48.5
+	winreg-0.50.0
+	xmlparser-0.13.5
+	zeroize-1.6.0
+"
+
+inherit coreos-cargo cros-workon systemd
+
+DESCRIPTION="Prototype Omaha Rust implementation"
+HOMEPAGE="https://github.com/coreos/ue-rs"
```
Comment (krnowak) [inline sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-9999.ebuild:204]:
```suggestion
HOMEPAGE="https://github.com/flatcar/ue-rs"
```

File: sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-0.1.0-r1.ebuild (hunk 1 above)
Comment (jepio) [reply sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-0.1.0-r1.ebuild:1]: i would keep this as is

**Review by t-lo — APPROVED** [review]
The ebuild looks sensible and tests run fine. LGTM.

File: sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-0.1.0-r1.ebuild (hunk 1 above)
Comment (krnowak) [reply sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-0.1.0-r1.ebuild:1]: `git blame` should be smart enough to follow renames. But I'm not going to block the PR on this.

**Review by krnowak — APPROVED** [review]

**Pull request merged**